package.json

```diff
@@ -13,10 +16,13 @@
     "test": "lerna link && COVERAGE=ON lerna run --stream --no-bail test",
     "prepare": "husky install",
     "reset-modules": "rm -rf node_modules/ packages/*/node_modules",
-    "publish-packages": "ts-node ./scripts/publish-packages.ts",
-    "deprecate-version": "ts-node ./scripts/deprecate-version.ts"
+    "publish-packages-script": "ts-node ./scripts/publish-packages.ts",
```

Comment:
this is the old script right? why not remove?

Reply:
If you see the commits, I did remove it initially, then I thought that perhaps we should keep it at least until we have tried out the new package a few times. If we need to deploy something and the package doesn't work, we'd have that fallback.